style.css

```diff
@@ -0,0 +1,52 @@
+.first-section {
+  border: 4px solid red;
+  font-size: 1.2rem;
+  margin-bottom: 20px;
+}
+
+.section-heading {
+  text-align: center;
+}
+
+.link {
+  color: rgb(197, 62, 186);
+}
+
+.button {
+  background-color: rgb(100, 180, 100);
+  height: 50px;
+  border: white;
+  width: 130px;
+  cursor: pointer;
+  color: white;
+}
+
+.section-title {
+  color: red;
+  font-size: 2rem;
+  margin: 1rem 200px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.section-part {
```

Comment:
Need to remove extra properties for the whole file.
CC: @nb-1231

Reply:
resolved

Reply:
I'm still able to see extra properties there is nothing changed from your side.